openapi-v3.yaml

```diff
@@ -724,16 +752,14 @@ paths:
               schema:
                 type: string
                 description: Raw file contents product by dbt
-                example:
-                  nodes: {}
-                  sources: {}
-                  macros: {}
-                  docs: {}
-                  disabled: []
-                  generated_at: "2020-01-01T12:00:00.00000Z"
-                  parent_map: {}
-                  child_map: {}
-
+                example: "# package.yaml
+                  packages:
+                  - package: dbt-labs/audit_helper
+                  version: 0.5.0
+                  - package: dbt-labs/codegen
+                  version: 0.5.0
+                  - package: dbt-labs/dbt_utils
+                  version: 0.8.2"
```

Comment:
Linting fix, I chose to use an example package.yml as it's potentially more tangible, but we could go back to using an empty manifest if desired.